app/views/pages/_signin.erb

```diff
@@ -0,0 +1,17 @@
+<div class='jumbotron row m-4 px-5 py-3'>
+  <%if @previous_course %>
+    <%= image_tag(@previous_course.image,
+        class: 'col-sm-4',
+        alt: '...') %>
+  <% else %>
+    <%= image_tag('https://a.storyblok.com/f/50602/350x247/dfebba3390/homepage_icons-03.png',
+        class: 'col-sm-4',
+        alt: '...') %>
```

Comment:
Would be good to add some `alt` values here, and above (for accessibility). For the course one above, you could set it to the course title.